### PR TITLE
doc: Update acrn-manager README and QoS introduction

### PR DESCRIPTION
**Pull request by lirui34**
Modify 'launch_UOS.sh' to 'launch_uos.sh' and provide an example for
the script which is support '-C' option.

Signed-off-by: ruix.li <ruix.li@intel.com>

### CODE REVIEW

File: doc/tutorials/acrn-dm_QoS.rst [hunk 1]
```diff
@@ -112,6 +112,10 @@ How to use ACRN-DM QoS
 
       # acrnctl add launch_uos.sh -C
 
+.. note:: You can get the `example
```
Comment (dbkinder) [inline doc/tutorials/acrn-dm_QoS.rst]: Let's change to:
```
.. note:: You can download an `example launch_uos.sh script 
   <https://raw.githubusercontent.com/projectacrn/acrnhypervisor/master/devicemodel/samples/nuc/launch_uos.sh>`_
   that supports the ``-C``  (``run_container`` function) option.
```

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: tools/acrn-manager/README.rst [hunk 2]
```diff
@@ -41,17 +41,21 @@ Add a VM
...
+   # acrnctl add launch_uos.sh -U 1
    vm1-14:59:30 added
 
 If a ``-C`` option is also specified, the VM is launched in a runC
 container::
 
-   # acrnctl add launch_UOS.sh -C
+   # acrnctl add launch_uos.sh -C
+
+.. note:: You can get the `example
```
Comment (dbkinder) [inline tools/acrn-manager/README.rst]: Same change here:
```
.. note:: You can download an `example launch_uos.sh script 
   <https://raw.githubusercontent.com/projectacrn/acrnhypervisor/master/devicemodel/samples/nuc/launch_uos.sh>`_
   that supports the ``-C``  (``run_container`` function) option.
```

@gvancuts was hoping you'd add more information about how the ``-C`` option is handled (for example, all the work is done in the ``launch_uos.sh`` script and not by the ``acrnctl`` command) and more about what running in a runC container means.

**Comment by dbkinder** [comment]
@gvancuts I'm told the launch_uos.sh script can't be updated directly because there's no time to do a QA run before the v1.1 release this week, so we're releasing this version as an example the user can manually install.

File: tools/acrn-manager/README.rst (hunk 2 above)
Comment (Ragnarliu) [reply tools/acrn-manager/README.rst]: Hi Dave,
   The "-C" parameter is handled by launch_uos.sh script,  but every time you add the VM by acrnctl add command you need tell acrnctl this VM need launched with "-C" parameter, so acrnctl need store the "-C" parameter of the VM.
   If a VM is added with "-C" parameter, every time you start the VM, a runC container will be started at first, then the acrn-dm will run in the container. You can get this message from the ACRN-dm QoS HLD diagram.

Thanks

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: tools/acrn-manager/README.rst (hunk 2 above)
Comment (gvancuts) [reply tools/acrn-manager/README.rst]: > The "-C" parameter is handled by launch_uos.sh script, but every time you add the VM by acrnctl add command you need tell acrnctl this VM need launched with "-C" parameter, so acrnctl need store the "-C" parameter of the VM.

Unless I missed something in the `acrnctl` code, all `acrnctl` really does is check if `runc` is present on the system. It then strips `-C` from the current list of arguments and adds it back so the `launch_uos.sh` script picks it up. But doesn't `acrnctl` preserve all arguments that are placed after the launch script anyway?

(And checking if `runc` is available on the system could very easily be done in the `launch_uos.sh` script itself (e.g. `if [ -x /sbin/runc]`)

> If a VM is added with "-C" parameter, every time you start the VM, a runC container will be started at first, then the acrn-dm will run in the container. You can get this message from the ACRN-dm QoS HLD diagram.

How about we add a link to that doc from here?

**Comment by gvancuts** [comment]
> @gvancuts I'm told the launch_uos.sh script can't be updated directly because there's no time to do a QA run before the v1.1 release this week, so we're releasing this version as an example the user can manually install.

OK, I don't believe we needed to update the script though, or am I missing something?

The thing to fix really is the fact we talk about `launch_UOS.sh` instead of `launch_uos.sh` but that is fixed now :-)

File: tools/acrn-manager/README.rst (hunk 2 above)
Comment (Ragnarliu) [reply tools/acrn-manager/README.rst]: Hi Geoffroy,
   We have updated a new version with the QoS link in here, Rui is working on it.
Thanks

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**